x-pack/plugins/security/public/account_management/account_management_page.tsx

```diff
@@ -23,7 +23,7 @@ export const AccountManagementPage: FunctionComponent = () => {
   const { services } = useKibana<CoreStart>();
 
   const currentUser = useCurrentUser();
-  const userProfile = useUserProfile<{ avatar: UserProfileAvatarData }>('avatar');
+  const userProfile = useUserProfile<UserProfileData>('*');
```

Comment:
Is there a particular reason you changed this to a wildcard? 

I appreciate that there's currently not an awful lot of data stored in user profiles but I think it's good practice to only return what's required to render the page.

Reply:
Fixed in latest commit!